draft-ietf-quic-recovery.md

```diff
@@ -348,6 +348,13 @@ acknowledgement delays are likely to be non-repeating and limited to the
 handshake. The endpoint can therefore use them without limiting them to the
 max_ack_delay, avoiding unnecessary inflation of the RTT estimate.
 
+Note however that a large acknowledgement delay can result in a substantially
+inflated smoothed_rtt, if there is either an error in the peer's reporting of
+the acknowledgement delay or in the endpoint's min_rtt estimate.  Therefore,
+prior to handshake confirmation, an endpoint can ignore RTT samples if adjusting
```

Comment:
Is this better as a MAY?  We never say you can ignore RTT samples in any other circumstance, so this feels a bit new.
```suggestion
prior to handshake confirmation, an endpoint MAY ignore RTT samples if adjusting
```